src/calling/haplotypes.rs

```diff
@@ -22,6 +22,15 @@ pub struct Caller {
     outcsv: Option<PathBuf>,
 }
 
+#[derive(Serialize)]
+struct Output {
+    haplotype_a: NotNaN<f64>,
+    haplotype_b: NotNaN<f64>,
+    haplotype_c: NotNaN<f64>,
```

Comment:
But we don't always have 3 haplotypes only. I think it would make most sense to use a hash map here. The problem is that I think the csv crate does not support the serialization of hashmaps or other kinds of maps yet. Hence, I'm sorry but I guess my suggestion was not the right idea. Instead, you should probably just write the records as Vec of the values in the right order.

Reply:
No problem, I will do that.